live-build/config/hooks/90-linux-migration-artifact.binary

```diff
@@ -150,13 +150,17 @@ if [[ -n "${DELPHIX_SIGNATURE_TOKEN:-}" ]] && [[ -n "${DELPHIX_SIGNATURE_URL:-}"
 	VERSION="5.3"
 	SIGN_URL="$DELPHIX_SIGNATURE_URL/upgrade/keyVersion/$VERSION/sign"
 
```

Comment:
Should we create a sub-shell and cd into `$DEPOT_DIRECTORY` like we do above?

Reply:
ugh, yeah ... I obviously copied over my testing code so none of this are in the `depot/` directory.

Reply:
We should test migration with the produced image